lib/tasks/notifications/hr-enrollment.js

```diff
@@ -0,0 +1,52 @@
+const moment = require('moment'),
+      Enrollment = require('../../models/enrollment'),
+      Company = require('../../models/company'),
+      isWorker        = process.env.CONTEXT === 'worker',
+      winston         = require('winston');
+
+let {
+  debug, error
+} = winston;
+
+const debugPfx = 'Task :: HR Enrollments ::';
+
+if (isWorker) {
+  require('../');
+  debug = console.log.bind(console);
+  error = console.error.bind(console);
+}
+
+module.exports = async function () {
+  let mailOptions = {
+    subject:  'Weekly notification for new enrollments',
+    template: 'hr_week-enrollments'
+  };
+
+  let companies = await Company.find({
+    benefitsEnabled: true,
+    $or:             [{
+      deactivatedOn: null
+    }, {
+      reactivatedOn: { $lte: new Date() }
+    }]
+  });
+
+
+  for (let company of companies) {
+    await Enrollment.find({
+      created:    { $gte: moment().subtract(7, 'days') },
```

Comment:
```suggestion
      created:    { $gte: moment().subtract(7, 'days').toDate() },
```

Not sure your test is testing this correctly. 😉